Source/items.cpp

```diff
@@ -5277,7 +5277,7 @@ void SpawnWitch(int lvl)
 
 	j = 3;
 #ifdef HELLFIRE
-	iCnt = random_(51, 15) + 10;
+	iCnt = random_(51, WITCH_ITEMS - 10) + 10;

```

Comment:
how about adding two defines? WITCH_ITEMS_MIN and WITCH_ITEMS_MAX ?

Reply:
Meh, then we would have to do the same for the smith.

Reply:
I think it'd be a good change : P otherwise why have max items at all

Reply:
The def sets the max and the logic varies by 10, but feel free to define both if you prefer